src/backend/src/utils/response.utils.ts

```diff
@@ -0,0 +1,65 @@
+import { Response } from 'express';
+
```

Comment:
[See this example](https://github.com/Northeastern-Electric-Racing/FinishLine/pull/447/files#r1023135726)